### PR TITLE
Add people recognition config

### PR DESCRIPTION
**Pull request by PetervDooren**
merge together with https://github.com/tue-robotics/robot_launch_files/pull/55

### CODE REVIEW

File: parameters/world_modeling/people_recognition_3d.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+heuristic: "shoulder"
+arm_norm_threshold: 0.5
+neck_norm_threshold: 0.7
+wave_threshold: 0.2
+vert_threshold: 0.7
+hor_threshold: 0.4
+padding: 5
+enable_topic_mode: false
+
+camera_info_depth: "hero/head_rgbd_sensor/depth_registered/camera_info"
```
Comment (MatthijsBurgh) [inline parameters/world_modeling/people_recognition_3d.yaml]: Absolute topics should start with a slash

Comment (PetervDooren) [reply parameters/world_modeling/people_recognition_3d.yaml]: True that

**Comment by LarsJanssenTUe** [comment]
@MatthijsBurgh Ping!

**Comment by MatthijsBurgh** [comment]
I don't think we made a final decision on the topic mode. Or did we @jlunenburg @PetervDooren?

**Comment by PetervDooren** [comment]
We did not

**Pull request closed without merging**